### DISCUSSION TITLE
How to Permanently Enable GPS Time Sync & 24 fps/Hz?

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#541] [Q&A] [answered]
I would like to enable these two Labs features and have them be set even after a reboot, shutdown or removal of the battery:
- [GPS Time Synchronization](https://gopro.github.io/labs/control/gpssync/)
- [24 Hz Mode](https://gopro.github.io/labs/control/set/?cmd=oM24HZ=1)

Would this be possible to do like how it's done in this example?:
[IMU Trigger](https://gopro.github.io/labs/control/imutrigger/)

EDIT 1: 
This leads me to ask how I would create the QR code with dual boot commands?

I.e. how would I fill out the ["Metadata Four CC" and "Metadata Value(s)"](https://gopro.github.io/labs/control/extensions/) fields to achieve the above mentioned goal?

I'm trying my best here but haven't figured it out yet.

EDIT 2:
Would this be it?:
```
https://gopro.github.io/labs/control/set/?cmd=!MBOOT=%22!MSYNC=1%22!MBOOT=%22oM24HZ=1%22
```

Or is it something in it that needs modifying?

**Comment by dnewman-gpsw** [comment]
No you don't need to use a boot command for this. Anything with !Mwxyz= is permanently set and is active on every boot.

**Reply by dnewman-gpsw** [reply]
So scan this once:
!MSYNC=1!M24HZ=1

**Reply by AlexLandherr** [reply, marked as answer]
So like this if I understand you correctly?:
[https://gopro.github.io/labs/control/set/?cmd=!MSYNC=1!M24HZ=1](https://gopro.github.io/labs/control/set/?cmd=!MSYNC=1!M24HZ=1)

EDIT: I added a title to it:
[https://gopro.github.io/labs/control/set/?cmd=!MSYNC=1!M24HZ=1&title=Permanent%20GPS%20Sync%20and%2024Hz](https://gopro.github.io/labs/control/set/?cmd=!MSYNC=1!M24HZ=1&title=Permanent%20GPS%20Sync%20and%2024Hz)

**Reply by AlexLandherr** [reply]
Thanks! Will try now.

**Reply by AlexLandherr** [reply]
I put it to the test by the following method:
1. Showed the linked QR code to the camera which worked.
2. Recorded a 20 second clip at 5.3K 16:9 which I then checked the true fps of with VLC media player which showed "24" which I understand means 24.000 fps.
3. Rebooted the camera.
4. Repeated step 2 and got the same result in VLC media player.

Thanks again for putting up with my beginner questions. It's highly appreciated!
